symengine/lib/symengine_wrapper.pyx

```diff
@@ -843,6 +885,160 @@ cdef class Constant(Basic):
             raise Exception("Unknown Constant")
 
 
+cdef class Boolean(Basic):
+    pass
+
+
+class BooleanAtom(Boolean):
+    pass
+
+
+class BooleanTrue(BooleanAtom):
+
+    def _sympy_(self):
+        import sympy
+        return sympy.S.true
+
+    def _sage_(self):
+        return True
+
+
+class BooleanFalse(BooleanAtom):
+
+    def _sympy_(self):
+        import sympy
+        return sympy.S.false
+
+    def _sage_(self):
+        return False
+
+
+class And(Boolean):
+
+    def __new__(cls, *args):
+        cdef symengine.set_boolean s
+        cdef Boolean e_
+        for e in args:
+            e_ = sympify(e)
+            s.insert(e_.thisptr)
```

Comment:
The builds fail because of the following error:
```
 Cythonizing symengine_wrapper.pyx
  
  Error compiling Cython file:
  ------------------------------------------------------------
  ...
  def logical_and(*args):
      cdef symengine.set_boolean s
      cdef Boolean e_
      for e in args:
          e_ = sympify(e)
          s.insert(e_.thisptr)
          ^
  ------------------------------------------------------------
  
  C:\projects\symengine-py-l1jmr\symengine\lib\symengine_wrapper.pyx:3043:9: Object of type 'set_boolean' has no attribute 'insert'
```